app/main/lib/shared_models/video_model.py

```diff
@@ -172,8 +192,8 @@ def search(self, task):
             if temporary:
                 self.delete(task)
```

Comment:
We should also try to explicitly delete `folder` and `tempfile` at this point if they exist (I don't see those updated values having been added to `task`

Reply:
`folder` here is `tmp` and `tempfile` is a tempfile - I can certainly explicitly delete the tempfile but I believe it is, by definition, unnecessary?